app/javascript/components/FiltersBox.js

```diff
@@ -68,7 +68,7 @@ export default function FiltersBox(props) {
 
   return (
     <div className={componentName} id={filtersBoxID} style={{display: props.show ? '' : 'none'}}>
-      <FiltersSearchBar filtersBoxID={filtersBoxID} />
+      <FiltersSearchBar filtersBoxID={filtersBoxID} facetID={props.facet.id} />
```

Comment:
I think it's more standard to capitalize names like this as facetId, but I don't feel too strongly about it

Reply:
Addressed in 3bbaa15.